examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -18,6 +19,18 @@ if (location.hash.length === 0) {
 const containerId = location.hash.substring(1);
 document.title = containerId;
 
+// Define a custom ITelemetry Logger
+class CustomLogger implements ITelemetryBaseLogger {
+    constructor() {}
+    send(event: ITelemetryBaseEvent) {
+        console.log("Custom telemetry object array: ".concat(JSON.stringify(event)));
+    }
+}
+
+// Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+// and hook to the Telemetry system
+const customLogger: CustomLogger = new CustomLogger();
```

Comment:
keep this scoped to its usage in `start()` below

Reply:
good point. I was just thinking to put it with other parameter objects like containerId and containerSchema.